apps/console/app/lib/env.ts

```diff
@@ -1,7 +1,9 @@
-export const isStackAuthEnabled = !!import.meta.env.VITE_STACK_PROJECT_ID;
 // isDevLocal is true only in pure local dev: no Turbo/CI context and no explicit service URLs
 export const isDevLocal =
   !process.env.TURBO_HASH &&
   !import.meta.env.VITE_API_URL &&
-  !import.meta.env.VITE_GATEWAY_URL;
+  !import.meta.env.VITE_GATEWAY_URL &&
+  !import.meta.env.VITE_AUTH_URL;
```

Comment:
Nitpick: Ordering 
1/ Either by stack depth auth -> api -> gateway
2/ Or alphabetically api -> auth -> gateway

(also in other places)